styles.css

```diff
@@ -136,3 +140,107 @@ body {
 .main__button:hover {
   opacity: .5;
 }
+
+@media (max-width:667px) {
+  #checkbox {
+    display: none;
+  }
+
+/* .container, .logo__circle {
+  margin: 15px 0;
+} */
+
+  .rectangle1 {
+    position: absolute;
+    width: 25px;
+    height: 3px;
+    left: 326px;
+    top: 32px;
+    background: #0056EC;
+  }
+
+  .rectangle2 {
+    position: absolute;
+    width: 25px;
+    height: 3px;
+    left: 326px;
+    top: 39px;
+    background: #0056EC;
+  }
+
+  .rectangle3 {
+    position: absolute;
+    width: 25px;
+    height: 3px;
+    left: 326px;
+    top: 46px;
+    background: #0056EC;
+  }
+
+  .logo__circle {
+    left: 24px;
+    top: 24px;
+    width: 36px;
+    height: 35px;
+    position: absolute;
+  }
+
+  .logo__circle_color_blue {
+    position: absolute;
+    width: 28px;
+    height: 28px;
+    left: calc(50% - 28px/2 - 141.5px);
+    top: 31px;
+    background: #C8DBFB;
```

Comment:
Codacy found an issue: [Expected "#C8DBFB" to be "#c8dbfb" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6356170)